ftp-ballerina/Package.md

```diff
@@ -1,214 +1,17 @@
-## Module Overview
+## Package Overview
 
-This module provides an FTP client and an FTP server listener implementation to facilitate an FTP connection to a remote location.
+The FTP library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
 
-###FTP Client
+It provides an implementation for performing CRUD operations on remote FTP servers with FTP protocol.
 
-The `ftp:Client` connects to an FTP server and performs various operations on the files. Currently, it supports the 
-generic FTP operations; `get`, `delete`, `put`, `append`, `mkdir`, `rmdir`, `isDirectory`, `rename`, `size`, and
- `list`.
+For more information go to [The FTP Package](https://docs.central.ballerina.io/ballerina/ftp/latest).
```

Comment:
shall we remove API doc link from here. in central, there is a link to API docs

Reply:
Removed.